src/Commands/BuildCommand.php

```diff
@@ -62,7 +68,7 @@ public function execute(InputInterface $input, OutputInterface $output)
         $this->themeName = $input->getArgument('theme');
 
         $currentPath = getcwd();
-        $config = require $currentPath.'/ibis.php';
+        $this->_config = require $currentPath.'/ibis.php';
```

Comment:
Maybe it's better to load that configuration in the constructor?

Reply:
I created a `__construct()` method that calls `super::__construct()` before assigning `$this->config`.

This change also required the `$currentPath` value in the constructor, so I moved it to another private field in the class and cleaned up all the method signatures and updated the all references to use `$this->currentPath` instead.